examples/pom.xml

```diff
@@ -14,6 +14,20 @@
             <artifactId>scalecube-services</artifactId>
             <version>${project.version}</version>
         </dependency>
+        
+        <!-- https://mvnrepository.com/artifact/org.rapidoid/rapidoid-http -->
+		<dependency>
+			<groupId>org.rapidoid</groupId>
+			<artifactId>rapidoid-quick</artifactId>
```

Comment:
Don't put such dependency into this repository. If you want to make such kind of high level example, make it in a separate repository